lumicks/pylake/kymo.py

```diff
@@ -212,6 +212,20 @@ def _to_spatial(self, data):
     def _shape(self):
         return (self.pixels_per_line,)
 
+    @property
+    def shape(self):
+        """Shape of the reconstructed `Kymo` image"""
+        # For a kymo the only way to find the shape is to perform the reconstruction. While one
+        # could traverse the info-wave for this purpose, we may as well just reconstruct the image
+        # since that still has the potential to be useful in the future (since it is cached).
```

Comment:
Yeah, I figured a reminder would be good. In the future, if we ever get the line count up front in the metadata, we can include a fast path to just pull it from metadata, because it's wasteful.